docs/adr/0004-multi-tenant-architecture.md

```diff
@@ -58,4 +58,4 @@ For the foreseeable future, the plan is to set up a total of three repositories:
 
 The biggest challenge is keeping repositories in sync. In order to be able to use configuration settings, the responsible configuration repositories have to contain the configuration before it can be used by `signals-frontend`.
 
-Another challenge is the deploy preview that is currently handled by Netlify; each branch in `signals-frontend` can be published separately to show-case specific features or functionality. However, since the code and the configuration is split, a `signals-frontend` build will contain the default configuration and not the instance-specific configuration and thus Netlify doesn't seem suitable at this point in time.
+~~Another challenge is the deploy preview that is currently handled by Netlify; each branch in `signals-frontend` can be published separately to show-case specific features or functionality. However, since the code and the configuration is split, a `signals-frontend` build will contain the default configuration and not the instance-specific configuration and thus Netlify doesn't seem suitable at this point in time.~~
```

Comment:
..that is currently handled by Netlify;.. -> Zou dat niet ..that was handled by Netlify;.. moeten zijn?

Reply:
It looks like this...
<img width="1022" alt="Screenshot 2023-09-18 at 20 12 30" src="https://github.com/Amsterdam/signals-frontend/assets/46756331/8b1d41d4-0335-46c5-8156-60d0119bc3f4">